cordova-lib/src/PlatformProject.js

```diff
@@ -257,11 +257,15 @@ function save() {
 PlatformProject.prototype.build = build;
 function build(opts) {
     var self = checkThis(this);
-    var bin = path.join(self.root, 'cordova', 'build');
-    var args = [];
 
-    var copts = { stdio: 'inherit' };
-    return superspawn.spawn(bin, args, copts);
+    return self.copyWww(opts.paths.www)
```

Comment:
Also, build probably shouldn't get the whole `prjInfo` metadata as I assume here.  If copyWww moves out, maybe there are no opts at all.